lib/matplotlib/backends/backend_svg.py

```diff
@@ -531,11 +676,25 @@ def _write_svgfonts(self):
 
     def open_group(self, s, gid=None):
         """
-        Open a grouping element with label *s*. If *gid* is given, use
-        *gid* as the id of the group.
+        Open a grouping element with label `s`. If `gid` is given, use
```

Comment:
These changes are unnecessary and not right. Sphinx cannot make references to the arguments, so single-backticks are incorrect. Our style guide says [use single-asterisks in this case](http://matplotlib.org/2.0.2/devel/documenting_mpl.html#formatting).